protos/flyteidl/service/admin.proto

```diff
@@ -376,4 +377,14 @@ service AdminService {
       description: "Retrieve input and output data from an existing task execution."
     };
   }
+
+  rpc UpdateProjectDomainAttributes (flyteidl.admin.ProjectDomainAttributesUpdateRequest) returns (flyteidl.admin.ProjectDomainAttributesUpdateResponse) {
```

Comment:
Do we have Create? or is Update == CreateOrUpdate? is that common?

Reply:
fwiw, In another PR for entity metadata, I'm following a create-or-update pattern with an UPDATE endpoint

Reply:
@EngHabu for this i didn't want to bother with create or update since the config'd defaults will always be applied and i didn't want to have to expose the underlying model to the end user. this does depart from the explicit create in our other models but since this has a default anyways i think it makes sense